package.json

```diff
@@ -31,12 +32,9 @@
     },
     "rules": {
       "shopify/jest/no-snapshots": "off",
-      "shopify/jest/no-vague-titles": [
-        "error",
-        {
-          "allow": "all"
-        }
-      ]
+      "@typescript-eslint/ban-ts-ignore": "off",
+      "@typescript-eslint/no-unnecessary-condition": "off",
+      "@typescript-eslint/no-misused-promises": "off"
```

Comment:
Were all of these so pervasive they needed to be disabled?

Reply:
Yeah, I'll add a comment to mention that.